docs/source/getting_started/writing_code.ipynb

```diff
@@ -0,0 +1,308 @@
+    "At least the basic include statements etc.\n"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Helpful Templates while developing Algorithms in MAAP\n",
+    "\n",
+    " - conda.yml with some default packages\n",
+    " - put some templates here, or link to them?"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Working with code repositories like GitHub and GitLab"
+   ]
+  },
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### The MAAP GitLab Code repository\n",
+    "\n",
+    "After creating your MAAP account, you can create a code repository by navigating to the MAAP GitLab account at https://repo.ops.maap-project.org. This GitLab account is connected to your ADE workspaces automatically when signing into the ADE.\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This example walks through cloning a repository into the ADE. Cloning a repository allows you to open, edit, and run files contained within the cloned repository. In this example, we look at cloning the \"MAAP-Project/maap-documentation\" Git repository, so that you are able to experiment with the code examples contained within this user documentation."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "When inside of a workspace, navigate to **Git** tab at the top of the Jupyter window. Click it to see the option to **Clone**."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "![Git Clone](../_static/git_clone.png) "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We can also access the \"Clone a repo\" dialogue box by selecting the **File Browser** ![File Browser](../_static/file_browser.png) tab on the JupyterLab sidebar and using the **Git Clone** ![Git Hub](../_static/git_hub.png) button located at the top of the sidebar. The \"Clone a repo\" dialogue box prompts you to enter the URI of the repository you wish to clone. For this example we enter \" https://github.com/MAAP-Project/maap-documentation.git \". This can be found by visiting the [GitHub site for the \"MAAP-Project/maap-documentation\" Git repository](https://github.com/MAAP-Project/maap-documentation) and clicking the **Code** button. ![Code Button](../_static/code_button.png) Then press the **CLONE** button to clone the repository into the ADE. ![Clone a Repo](../_static/clone_a_repo.png)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "With the **File Browser** tab on the JupyterLab sidebar selected, a folder named \"maap-documentation\" should now appear under the \"projects\" directory. Folders for the various sections of the guide can be found in the \"docs/source/\" directory. "
```

Comment:
Do we want to say home directory instead of "projects" directory to be more consistent with the naming?

Reply:
The creating ssh keys section, it is referred to as the home directory. Is that incorrect? 
Also, this section says home directory: 
"NOTE: Custom environments are deleted, and modifications to the base conda environment (e.g., installing mamba) are reset, when the workspace is Stopped. To preserve a custom environment after the workspace shutdowns, create the environment that is saved to the home directory instead of the default system directory for environments. Make sure to not make a conda env inside a git-tracked folder, or if you do add it to the `.gitignore`. If git is tracking an env, it could cause your workspace to crash. This can be done via the following commands, to create the environment at the path `./myenvs/testenv` with the flag `-p` and to activate it using the full path:\n",

Reply:
by "home" I usually think of "~"

Reply:
sure but on a normal *nix system that ~ would actually go to `/home/username` MAAP is very non-standard in this way so it's somewhat confusing.

Reply:
How about: "Your Jupyter home directory (~) is mounted to /projects. Files in here persist across sessions."